recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,260 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os, shutil
+
+class AwsSdkCppConan(ConanFile):
+    name = "aws-sdk-cpp"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aws/aws-sdk-cpp"
+    description = "AWS SDK for C++"
+    topics = ("aws", "cpp", "crossplateform", "amazon", "cloud")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    sdks = ("access-management",
```

Comment:
Do you foresee build problems with some optional modules?

Reply:
Space found one at least https://github.com/conan-io/conan-center-index/pull/4401#discussion_r598333244